ui-components/src/locales/general.json

```diff
@@ -1032,7 +1032,14 @@
       "individualUnits": "Individual Units",
       "delete": "Delete this Unit",
       "deleteConf": "Do you really want to delete this unit?",
-      "eligibility": "Eligibility"
+      "eligibility": "Eligibility",
+      "typeOptions": {
```

Comment:
These translations still exist elsewhere under `preferredUnit` I think we can delete these added translations and use the others yeah?

Reply:
@dominikx96 , can you please make this update.

Reply:
This also existed in listings.unitTypes. I think the options for preferredUnit are a little different and we may not want a one to one here, since those have a <number> + option. So I opted to keep the new entries and remove the older listings.unitTypes.